PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -191,6 +195,9 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
     }
 
     func filterByTagAction() -> UIAction? {
+        let event = SnowplowEngagement(type: .general, value: nil)
+        let contexts: [Context] = [UIContext.home.screen, UIContext.myList.tagBadge]
```

Comment:
I need to double check if `home` is the right value. I think `my_list` makes more sense, but let me confirm with Sreejita on this.